app/main/posts/detail/post-messages.directive.js

```diff
@@ -40,7 +40,6 @@ function (
             $scope.totalItems = $scope.itemsPerPage;
             $scope.pageChanged = getMessagesForPagination;
             $scope.showPagination = false;
-            $scope.isLoadingMessages = false;
```

Comment:
Removing the loading bar for messages since there already is a loading-bar in the toolbar = 2 loading bars in the same view = confusing.